### PR TITLE
Add 'oblicubes'

### PR DESCRIPTION
**Pull request by trevorld**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1982,7 +1982,7 @@ widgets:
     jslibs: >
     ghuser: piecepackr
     ghrepo: piecepackr
-    tags: board games
+    tags: board games, geoms
```
Comment (yutannihilation) [inline _config.yml:1985]: Could you remove this diff?

**Comment by trevorld** [comment]
> Could you remove this diff?

I'm also the author/maintainer of `{piecepackr}` which implements the geom function `geom_piece()`.  Should this tiny change to its tags be a separate pull request?

**Comment by yutannihilation** [comment]
Oh, I see. As you didn't describe anything about it here and the github repo is not under your username, I thought you just mistyped it.

**Comment by yutannihilation** [comment]
Thanks!

**Pull request merged**